source/release-notes/1.8.txt

```diff
@@ -21,9 +21,83 @@ Release Notes for mongosync 1.8
 
 **Upcoming**
 
-New Features:
+Updates to ``mongosync`` ``/progress`` API endpoint
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Returns a top-level boolean ``success`` field.
+
+- Returns the integer ``totalEventsApplied`` field, which approximates 
+  the number of change events applied to a destination cluster by an 
+  instance of ``mongosync``.
+
+- Reports per-partition progress tracking with the following changes:
+
+  - Calls to ``/progress`` before ``mongosync`` reaches the collection 
```

Comment:
I think we should move the details (the 4 sub-bullets) to the /progress endpoint page. And this can be summarized here in the release notes with just `Reports per-partition progress tracking`

Reply:
Moved to the "Behaviors" section of /progress in DOCSP-41853. ([direct link to file diff](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/362/files#diff-ab8a67f75f38f7f53de310897e90928abeb453eadbd1632ee35d2468147956bc))